public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
 
-# When Developing
+There are many Ethereum clients to choose from. We recommend different clients depending on whether you are developing or deploying.
 
-* EthereumJS TestRPC: [https://github.com/ethereumjs/testrpc](https://github.com/ethereumjs/testrpc)
+## When developing
 
-When developing your Truffle-based application, we recommend using the [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). It's a complete blockchain-in-memory that runs only on your development machine. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90%.
+### Truffle Develop
 
-# When Deploying to Live Networks
+We recommend using Truffle Develop, a development blockchain built directly into Truffle. It processes transactions instantly instead of waiting for the default block time -- so you can test that your code works quickly -- and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing, and Truffle knows how to use its special features to speed up test runtime by almost 90% compared to other clients.
+
+Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following into a terminal:
+
+```shell
+truffle develop
+```
+
+This will run the client on `http://localhost:9545`. It will display the first 10 accounts and the mnemonic used to create those accounts. ([Read more about account mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).) Truffle Develop uses the same mnemonic every time to make developing your applications as easy as possible:
+
+```
+candy maple cake sugar pudding cream honey rich smooth crumble sweet treat
+```
+
+Once launched, Truffle Develop will provide you with a console you can use to run all available Truffle commands. These commands are input by omitting the `truffle` prefix. So, for example, to compile your smart contracts, instead of typing `truffle compile`, you need to only type `compile`.
+
+### TestRPC
+
+You can also use [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). It's a separate command-line blockchain you can use for both development and automated testing, and works well with Truffle. Check out its [README](https://github.com/ethereumjs/testrpc/blob/master/README.md) and documentation for more information.
+
+To run the TestRPC, open a new terminal and type:
+
+```shell
+testrpc
+```
+
+The TestRPC will start with a different seed each time (and so a different mnemonic and account list), but you can run the TestRPC with a specific mnemonic in order to regenerate the same accounts:
+
+```shell
+testrpc --mnemonic word1 word2 word3 ...
+```
+
```

Comment:
We should take out lines 28 through 39 here as we want to deemphasize its usage.